ui/lib/pki/addon/components/page/pki-role-details.hbs

```diff
@@ -52,15 +52,7 @@
             @label={{capitalize (or attr.options.detailsLabel attr.options.label (humanize (dasherize attr.name)))}}
             @value={{val}}
             @alwaysRender={{true}}
-          >
-            {{#if (gt val.length 0)}}
-              {{#each val as |key|}}
-                <span>{{key}}, </span>
-              {{/each}}
-            {{else}}
-              None
-            {{/if}}
-          </InfoTableRow>
+          />
```

Comment:
this goes to type `array` in InfoTableRow which handles adding the comma (except for after the last item in the list)